packages/debugger/test/helpers.js

```diff
@@ -134,38 +128,18 @@ export async function defaultMigrations(contractNames) {
 }
 
 export async function compile(config) {
-  const {compilations} = await WorkflowCompile.compileAndSave(
+  const { compilations } = await WorkflowCompile.compileAndSave(
     config.with({
       all: true,
       quiet: true
     })
   );
-  const collectedCompilationOutput = compilations.reduce(
-    (a, compilation) => {
-      if (compilation.compiler.name === "solc") {
-        for (const contract of compilation.contracts) {
-          a.contractNames = a.contractNames.concat(contract.contractName);
-        }
-        a.sourceIndexes = a.sourceIndexes.concat(compilation.sourceIndexes);
-      }
-      return a;
-    },
-    {contractNames: [], sourceIndexes: []}
-  );
-  const sourceIndexes = collectedCompilationOutput.sourceIndexes.filter(
-    (item, index) => {
-      return collectedCompilationOutput.sourceIndexes.indexOf(item) === index;
-    }
+  const contractNames = flatten(
+    compilations.map(compilation =>
+      compilation.contracts.map(contract => contract.contractName)
+    )
   );
-  const contractNames = collectedCompilationOutput.contractNames.filter(
-    (item, index) => {
-      return collectedCompilationOutput.contractNames.indexOf(item) === index;
-    }
-  );
-  return {
-    contractNames,
-    files: sourceIndexes
-  };
+  return { compilations, contractNames };
```

Comment:
Do you not need to adjust whatever code was previously calling this method? Or maybe it currently only uses `contractNames`?

Reply:
Oh nevermind, I see that you already did adjust it in this file!

Reply:
Yup, your two comments answer one another. :)